mods/ts/rules/ai.yaml

```diff
@@ -15,6 +15,7 @@ Player:
 		MaximumExcessPower: 200
 		ExcessPowerIncrement: 30
 		ExcessPowerIncreaseThreshold: 4
+		BaseSpacing: true
```

Comment:
I tested this change but the AI did this:
![grafik](https://user-images.githubusercontent.com/7704140/125644413-bb7a7592-6f66-42e5-b2af-711eb8820f3e.png)

So this might not be working properly for isometric mods?

Reply:
I think it may not have worked at all as I was testing against top left instead of the whole footprint.